src/event_data.cc

```diff
@@ -172,8 +172,17 @@ namespace grpc_labview
             // part of its FINISH state.
             new CallData(_server, _service, _cq);
 
-            _stream.Read(&_rb, this);
-            _status = CallStatus::Process;
+            auto name = _ctx.method();
+            if (_server->HasRegisteredServerMethod(name) || _server->HasGenericMethodEvent())
+            {
+                _stream.Read(&_rb, this);
+                _status = CallStatus::Process;
+            }
+            else
+            {
+                _status = CallStatus::Finish;
+                _stream.Finish(grpc::Status(grpc::StatusCode::UNIMPLEMENTED, ""), this);
```

Comment:
Q: Do we advertise the grpc::status somehow? when we say UNIMPLEMENTED will user see that?

if we are showing, then we may want to pass some error message like grpc::Status(grpc::StatusCode::UNIMPLEMENTED, "Method not implemented yet");

Reply:
This error status will make its way back to a LV error cluster and you will see the error below.
![image](https://user-images.githubusercontent.com/35128660/197774471-b21cc038-e0bc-4820-92cb-5e66401c973c.png)